static-assets/components/cstudio-dashboard-widgets/lib/wcm-dashboardwidget-common.js

```diff
@@ -1692,15 +1692,27 @@ WcmDashboardWidgetCommon.refreshDashboard = function(inst) {
   );
 };
 
+const dashboardsIds = ['MyRecentActivity', 'recentlyMadeLive', 'approvedScheduledItems', 'GoLiveQueue'];

```

Comment:
Don't these come from some XML file? We should use what comes from config somehow.

Reply:
@rart no, right now the dashboard js files are being imported in the site-dashboard.ftl file, not being retrieved from a config.